java/simulator/src/main/java/com/hazelcast/simulator/coordinator/DeploymentPlan.java

```diff
@@ -75,9 +80,11 @@ public DeploymentPlan addToPlan(int workerCount, String workerType) {
         for (int i = 0; i < workerCount; i++) {
             WorkersPerAgent workersPerAgent = nextAgent(workerType);
             AgentData agent = workersPerAgent.agent;
-            WorkerParameters workerParameters = driver.loadWorkerParameters(workerType, agent.getAddressIndex());
-            workersPerAgent.registerWorker(workerParameters);
+            WorkerParameters workerParameters = new WorkerParameters();
+            workerParameters.setAll(properties);
+            workerParameters.set("WORKER_TYPE", workerType);
```

Comment:
Very nitty pick, you have changed some of the other properties to have a lower case instead. I.e. in the case of simulator properties. If you are changing those, I wonder why not these too?

Reply:
My idea is that properties introduced by the Simulator remain upper case. To reduce the chance of conflicts.